test-framework/common/src/main/java/io/quarkus/test/common/QuarkusTestResource.java

```diff
@@ -23,6 +23,13 @@
      */
     Class<? extends QuarkusTestResourceLifecycleManager> value();
 
+    /**
+     * @return The arguments to be passes to the {@code QuarkusTestResourceLifecycleManager}
+     *
+     * @see QuarkusTestResourceLifecycleManager#init(String[])
+     */
+    String[] args() default {};
```

Comment:
Should it be args or initParameters si that what it does is extra clear? Open subject, really, just asking.

Reply:
Good idea, I'll do it!

Reply:
Fixed